### ISSUE TITLE
the inference results are worse when finetuned fully with own data?

### ISSUE THREAD

**Issue by runzeer** [a-r-r-o-w/cogvideox-factory#87]
### System Info / 系統信息

I wrote the I2V fully finetune training script according to your T2V script and trained for 8000 steps but the inference results are worse.

https://github.com/user-attachments/assets/985cc0ca-bb1f-4486-a549-d0200bf0678d

and the loss curve is as below:

![2024-11-13 16-26-20屏幕截图](https://github.com/user-attachments/assets/d1e331a6-6399-413a-9a2d-6d9d94cedb47)

I want to know if there are any issues wrong.

### Information / 问题信息

- [ ] The official example scripts / 官方的示例脚本
- [X] My own modified scripts / 我自己修改的脚本和任务

### Reproduction / 复现过程
training code

[cogvideox_image_to_video_sft.txt](https://github.com/user-attachments/files/17728339/cogvideox_image_to_video_sft.txt)

inference code

[sample_video_shutter_stock_c2_finetuned_ckpt.txt](https://github.com/user-attachments/files/17728345/sample_video_shutter_stock_c2_finetuned_ckpt.txt)

### Expected behavior / 期待表现

Could you help me solve the issues?

**Comment by sayakpaul** [comment]
Is it possible to leverage the existing I2V training script? 
https://github.com/a-r-r-o-w/cogvideox-factory/blob/main/training/cogvideox_image_to_video_lora.py
